src/modules/tasksCreator.js

```diff
@@ -0,0 +1,64 @@
+const dynamicSort = (property) => {
+  const sortOrder = 1;
+  // if(property[0] === "-") {
+  //   sortOrder = -1;
+  //   property = property.substr(1);
+  // }
+  return (a, b) => {
+    let result = 0;
+    if (a[property] < b[property]) result = -1;
+    else if (a[property] > b[property]) result = 1;
+    return result * sortOrder;
+  };
+};
+export default class Tasks {
+  constructor() {
+    if (localStorage.getItem('tasks')) { this.list = JSON.parse(localStorage.getItem('tasks')); } else this.list = [];
+  }
+
+  populateStorage() {
+    localStorage.setItem('tasks', JSON.stringify(this.list));
+  }
+
+  add(description) {
+    const task = {
+      description,
+      completed: false,
+      index: this.list.length + 1,
+    };
+    this.list.push(task);
+    this.list.sort(dynamicSort('index'));
+    this.populateStorage();
+  }
```

Comment:
- [ ] Please I am unable to add a new task. Please kindly fix the issue since it is a requirement for this project.